mu/modes/pico.py

```diff
@@ -40,6 +40,7 @@ class PicoMode(ESPMode):
     valid_boards = [
         # VID  , PID,    Manufacturer string, Device name
         (0x2E8A, 0x0005, None, "Raspberry Pi Pico"),
+        (0x16d0, 0x08c7, None, "Tiny 2040")
```

Comment:
As far as I understand this VID/PID only appears with CircuiPython, so we should remove this entry from this mode.